conf/config.go

```diff
@@ -122,40 +122,49 @@ func LoadConfig(mainConfigFile string, fallbackConfigFile string) (*MenderConfig
 		return nil, loadErr
 	}
 
+	log.Debugf("Loaded %d configuration file(s)", filesLoadedCount)
+
 	if filesLoadedCount == 0 {
 		log.Info("No configuration files present. Using defaults")
 		return config, nil
 	}
 
-	if config.Servers == nil {
-		if config.ServerURL == "" {
+	log.Debugf("Loaded configuration = %#v", config)
+
+	return config, nil
+}
+
+// VerifyServerConfig verifies the Servers fields in the configuration
+func (c *MenderConfig) VerifyServerConfig() error {
+	if c.Servers == nil {
+		if c.ServerURL == "" {
 			log.Warn("No server URL(s) specified in mender configuration.")
 		}
-		config.Servers = make([]client.MenderServer, 1)
-		config.Servers[0].ServerURL = config.ServerURL
-	} else if config.ServerURL != "" {
+		c.Servers = make([]client.MenderServer, 1)
+		c.Servers[0].ServerURL = c.ServerURL
+	} else if c.ServerURL != "" {
 		log.Error("In mender.conf: don't specify both Servers field " +
 			"AND the corresponding fields in base structure (i.e. " +
 			"ServerURL). The first server on the list on the" +
```

Comment:
Would be nice if you can correct the typo "... The first server on the list ~~on the list~~ ..."